pkg/bootstrap/render.go

```diff
@@ -398,29 +339,16 @@ func convertKubeConfigSecrets(ctx context.Context,
 			return nil, fmt.Errorf("kubeconfig key not found in secret %s", s.Name)
 		}
 
-		bootstrapKubeConfigSecrets = append(bootstrapKubeConfigSecrets, BootstrapKubeConfigSecret{
-			Name:       s.Name,
-			KubeConfig: base64.StdEncoding.EncodeToString(secret.Data["kubeconfig"]),
+		bootstrapKubeConfigSecrets = append(bootstrapKubeConfigSecrets, chart.BootStrapKubeConfig{
+			Name: s.Name,
+			//	KubeConfig: base64.StdEncoding.EncodeToString(secret.Data["kubeconfig"]),
```

Comment:
delete? should this be encoded?

Reply:
deleted. 
the data is not needed to encode here since the helm chart has encoded the data.